src/ruby.rs

```diff
@@ -1450,4 +1487,38 @@ tests! {
         assert.rs_eq("Module", module.to_string_lossy());
         assert.rs_eq("Object", instance.to_string_lossy());
     }
+
+    #[test]
+    fn test_raise(assert: &mut Assertions) {
+        extern "C" fn __test_raise__(_self: VALUE) -> VALUE {
+            unsafe {
+                let prisvalue =  CStr::from_ptr(PRIsVALUE).to_string_lossy();
+                let format = format!("error: %{} (%+{}) failed", prisvalue, prisvalue);
+                let foo = "foo".to_ruby();
+                rb_raise(rb_eRuntimeError, CString::new(format).unwrap().as_ptr(), foo, foo);
+            }
```

Comment:
this is no good, see 
https://github.com/tildeio/libcruby-sys/pull/57#discussion_r217492950 (the comment in the code example)

Reply:
Yeah, I agree that it's problematic. Basically just did this for now to show that `PRIsVALUE` works. Probably need to expose a Ruby `sprintf` function or something.